scripts/ampHtmlValidator/index.test.js

```diff
@@ -91,6 +93,8 @@ describe('amp validator tests', () => {
   });
 
   it('should print passes when verbose is true', async () => {
+    jest.setTimeout(10000);
```

Comment:
Does this test really have much value, it just checks the validator logs and doesn't even check content of the logs? I wonder if we should improve it or delete it?

Reply:
Good point @andrewscfc, I think this test could be replaced by tests that check for 0 and non-zero exit codes for pass and fail respectively? Would surely be better than checking the logs

Reply:
Yeah, just having a 10 second test in our suite with little value seems a bit silly

Reply:
Agreed, on it 👍

Reply:
I added some tests which check exit codes and pass/fail status for valid and invalid amp html, but when running `yarn test:unit` they all timeout, so I I'll just delete the original flakey test.